src/services/connect.ts

```diff
@@ -0,0 +1,36 @@
+import { StellarSDK } from '../api/stellarSdk';
+import type { Keypair } from 'stellar-sdk';
+
+const encryptKey = (key: string, salt: number) => {
```

Comment:
the salt should not be passed in as a parameter.

The `encrypt` and `decrypt` functions should exist in a utils/security.ts file rather than live in the `connect` service